docs/releases/4.2.txt

```diff
@@ -236,6 +236,10 @@ Models
 * :class:`~django.db.models.functions.Now` now supports microsecond precision
   on MySQL and millisecond precision on SQLite.
 
+* ``Model`` now provides an asynchronous interface for some model methods that
+  user database. These are named as -per the existing synchronous operations
+  but with an ``a`` prefix, for example ``asave()``, ``adelete()``, and so on.
```

Comment:
The `QuerySet` release note in 4.1 didn’t list every method because it would have been too long. But we’re only adding three methods here, so I think we should list them. Also we should use `:meth:` for references so they work as links, which is a lot more useful for readers.

I suggest:

```suggestion
* ``Model`` now provides asynchronous versions of some model methods that use
  the database, using an ``a`` prefix: :meth:`~Model.adelete`,
  :meth:`~Model.arefresh_from_db`, and :meth:`~Model.asave`.
```

Reply:
Thanks for the review @adamchainz , but the ref is missing for these methods.